src/plugins/data/server/ui_settings.ts

```diff
@@ -705,8 +705,22 @@ export function getUiSettings(): Record<string, UiSettingsParams<unknown>> {
       }),
       schema: schema.boolean(),
     },
-    [UI_SETTINGS.DATAFRAME_HYDRATION_STRATEGY]: {
-      name: i18n.translate('data.advancedSettings.dataFrameHydrationStrategyTitle', {
+    [UI_SETTINGS.QUERY_ENHANCEMENTS_ENABLED]: {
+      name: i18n.translate('data.advancedSettings.query.enhancements.enableTitle', {
+        defaultMessage: 'Enable query enhancements',
+      }),
+      value: false,
```

Comment:
this still needs user to set `data.enhancements.enabled` in osd config yml, would it be confusing if user only sets it to true here and see enhancements are still disabled because yml is not set?